content/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/aggregatewindow.md

```diff
@@ -105,14 +111,14 @@ from(bucket: "example-bucket")
     fn: mean
   )
 ```
-###### Specify parameters of the aggregate function
+##### Specify parameters of the aggregate function
```

Comment:
Is the anonymous function part of the example? How to tell the difference between this example and the previous example (w/o an anonymous function)?

Reply:
Yes, the anonymous function is in the `fn` parameter of `aggregateWindow`. I updated the examples to use a data variable so the focus of the examples is on the `aggregateWindow` function.